src/components/Card.ts

```diff
@@ -1,57 +1,55 @@
 import styled, { css } from 'styled-components';
 import { media } from 'components/breakpoints';
+import { GU } from 'components/theme';
 
 export const Card = styled.div`
-    backdrop-filter: blur(10px);
-    background: rgba(255, 255, 255, 0.6);
-    border: 3px solid #000000;
-    box-sizing: border-box;
-    margin: 0 auto;
-    min-height: 10rem;
-    width: 30rem;
+  backdrop-filter: blur(${3 * GU}px);
+  background: rgba(255, 255, 255, 0.6);
+  border: ${GU}px solid #000000;
+  box-sizing: border-box;
+  margin: 0 auto;
+  min-height: ${25 * GU};
+  width: ${80 * GU}px;
 
-    ${media.xs`
-        border: 5px solid #000000;
-        width: 48rem;
+  ${media.xs`
+        width: ${120 * GU}px;
```

Comment:
Just to double check—is this right? 480px seems a bit big for most phones

Reply:
Yeah, fair enough. I've tended to just stick with conventional breakpoints: https://uiuxlab.medium.com/the-most-used-responsive-breakpoints-in-2017-of-mine-9588e9bd3a8a

Could lower to 448px if you think that's more appropriate?

Reply:
Don't worry, tested the app and looks fine 👍